pages/dkp/kommander/2.2/release-notes/2.2.0/index.md

```diff
@@ -71,42 +71,39 @@ This new predictive analytics tool provides greater support productivity, speed,
-| project-grafana-loki                           | 0.33.1  |                                                                                                                           |
-| project-logging                                | 1.0.0   |                                                                                                                           |
-| prometheus-adapter                             | 2.11.1  |                                                                                                                           |
-| reloader                                       | 0.0.99  |                                                                                                                           |
-| thanos                                         | 0.4.5   |                                                                                                                           |
-| traefik                                        | 10.3.0  |                                                                                                                           |
-| traefik-forward-auth                           | 0.3.2   |                                                                                                                           |
-| velero                                         | 3.1.1   |                                                                                                                           |
+| Platform Application | Version | Component Versions |
+|--------------------- |---------|--------------------|
+| cert-manager | 1.7.1 | - chart: 1.7.1<br>- cert-manager: 1.7.1 |
+| dex | 2.9.14  | - chart: 2.9.14<br>- dex: 2.22.0 |
+| external-dns | 6.1.8  | - chart: 6.1.8<br>- external-dns: 0.10.2 |
+| fluent-bit | 0.19.20 | - chart: 0.19.20<br>- fluent-bit: 1.8.13 |
+| gatekeeper | 3.7.0 | - chart: 3.7.0<br>- gatekeeper: 3.7.0 |
+| grafana-logging | 6.22.0 | - chart: 6.22.0<br>- grafana: 8.3.6 |
+| grafana-loki | 0.33.2 | - chart: 0.33.1<br>- loki: 2.2.1  |
+| istio | 1.11.6 | - chart: 1.11.6<br>- istio: 1.11.5 |
+| jaeger | 2.29.0  | - chart: 2.29.0<br>- jaeger: 1.31.0 |
+| karma | 2.0.1 | - chart: 2.0.1<br>- karma: 0.88 |
+| kiali | 1.47.0 | - chart: 1.47.0<br>- kiali: 1.47.0 |
+| knative | 0.3.9 | - chart: 0.3.9<br>- knative: 0.22.3 |
+| kube-oidc-proxy | 0.3.1 | - chart: 0.3.1<br>- kube-oidc-proxy: 0.3.0 |
+| [kube-prometheus-stack][kube-prometheus-stack] | 33.1.5  | - chart: 33.1.5 <br>- prometheus-operator: 0.54.1<br>- prometheus: 2.33.4<br>- prometheus alertmanager: 0.23.0<br>- grafana: 8.3.6 |
+| kubecost | 0.23.3 | - chart: 0.23.3<br>- cost-analyzer: 1.91.2 |
+| kubefed | 0.9.1 | - chart: 0.9.1<br>- kubefed: 0.9.1 |
+| kubernetes-dashboard | 5.1.1 | - chart: 5.1.1<br>- kubernetes-dashboard: 2.4.0 |
+| kubetunnel | 0.0.11 | kubetunnel: 0.0.11<br>- kubetunnel: 0.0.11 |
```

Comment:
@gracedo Not sure, but I think this should be chart + version instead of having kubetunnel two times? And a dash is missing.
```suggestion
| kubetunnel | 0.0.11 | -chart: <!-- xxx --> <br>- kubetunnel: 0.0.11 |
```

Reply:
Kubetunnel is our component; the chart version and component version are in lockstep.
c.f.: https://github.com/mesosphere/kubetunnel/releases/tag/v0.0.11

```suggestion
| kubetunnel | 0.0.11 | - chart: 0.0.11<br>- kubetunnel: 0.0.11 |
```